geant4.sh

```diff
@@ -6,13 +6,14 @@ build_requires:
   - CMake
 env:
   G4INSTALL: "$GEANT4_ROOT"
-  G4INSTALL_DATA: "$GEANT4_ROOT/share/Geant4-10.1.2"
+  G4INSTALL_DATA: "$GEANT4_ROOT/share/Geant4-10.1.3"
```

Comment:
See issue 289. The old value of $GINSTALL_DATA did not point to a valid location.